lib/tasks/api_action.rake

```diff
@@ -0,0 +1,11 @@
+namespace :api_action do
+  desc "subdomain analytics and report tasks"
+  
+  task :rerun_failed_actions => [:environment] do 
+    p "starting rerun_failed_actions task  @ #{Time.now}"
+      ApiAction.where(lifecycle_stage: 'failed', action_type: ['send_email', 'send_web_request']).each do |action|
```

Comment:
@Pralish you need to specify `tenant` here. Otherwise this will only run on the public schema. Rake tasks are global to the system.

Reply:
@Pralish  see more about accessing all tenants in Rake tasks here: https://github.com/restarone/violet_rails/blob/master/lib/tasks/report.rake